content/_data/changelogs/lts.yml

```
@@ -3604,6 +3604,311 @@
     message: |-
       Users with extended read permission now get a more read-only looking UI.
 
+- version: "2.235.1"
+  date: 2020-06-17
+  lts_predecessor: "2.222.4"
+  lts_baseline: "2.235"
+
+  changes: # compared to lts_baseline 2.235
+
+  - type: major bug
+    category: regression
+    pull: 4713
+    issue: 62163
+    authors:
+    - daniel-beck
+    message: |-
+      Make plugin manager work on Internet Explorer 11 again (regression in 2.231).
+  - type: major bug
+    category: regression
+    pull: 4712
+    issue: 61920
+    authors:
+    - MRamonLeon
+    message: |-
+      Prevent telemetry warnings about missing <code>javax.annotation</code> classes when running with Java 11 (regression in 2.231).
+  - type: major bug
+    category: regression
+    pull: 4714
+    issue: 62181
+    authors:
+    - jglick
+    message: |-
+      Fix a deadlock involving custom loggers during agent startup (regression in 2.231).
+  - type: bug
+    category: bug
+    pull: 4718
+    issue: 62231
+    authors:
+    - MRamonLeon
+    message: |-
+      Prevent Old Data Monitor from failing plugin loading in the case of class field unmarshalling issues.
+
+  lts_changes: # compared to lts_predecessor 2.222.4
+
+  # Major user experience improvements
+  - type: major rfe
+    category: major rfe
+    pull: 4546
+    authors:
+    - daniel-beck
+    message: |-
+      Organize entries on the <em>Manage Jenkins</em> page into categories and show them in a grid.
+  # - type: rfe
+  #   category: rfe
+  #   pull: 4586
+  #   authors:
+  #   - daniel-beck
+  #   message: |-
+  #     Add section headers to context menu of <em>Manage Jenkins</em>.
```

I like this particular change but wasn't sure if it should be included in the changelog
```suggestion
  - type: rfe
    category: rfe
    pull: 4586
    authors:
    - daniel-beck
    message: |-
      Add section headers to context menu of <em>Manage Jenkins</em>.
```